fastlane/Fastfile

```diff
@@ -601,6 +618,9 @@ end
     set_info_plist_value(path: "ownCloud Share Extension/Info.plist", key: "CFBundleDisplayName", value: "Share to " + appName)
     set_info_plist_value(path: "ownCloud Share Extension/Info.plist", key: "CFBundleName", value: appName)
 
+    set_info_plist_value(path: "ownCloud Action Extension/Info.plist", key: "CFBundleDisplayName", value: "Save to " + appName)
+    set_info_plist_value(path: "ownCloud Action Extension/Info.plist", key: "CFBundleName", value: appName)
+
```

Comment:
`ownCloud` also needs to be replaced with `appName` in the `CFBundleDisplayName` translations in `ownCloud Action Extension/*.lproj/InfoPlist.strings` (maybe using `sed` for search/replace), as the Info.plist contents for `CFBundleDisplayName` will otherwise be ignored and replaced with f.ex. `Speichern in ownCloud` again.

Reply:
fixed